packages/components/src/Badge/Badge.stories.js

```diff
@@ -23,6 +25,61 @@ const columnStyle = {
 const icons = {
 	'talend-cross': talendIcons['talend-cross'],
 	'talend-clock': talendIcons['talend-clock'],
+	'talend-caret-down': talendIcons['talend-caret-down'],
+	'talend-search': talendIcons['talend-search'],
+};
+
+const dropdownProps = {
+	id: 'context-dropdown-related-items',
+	label: 'Label',
+	items: [
+		{
+			id: 'context-dropdown-item-document-1',
+			label: 'document 1',
+			'data-feature': 'actiondropdown.items',
+			onClick: action('document 1 click'),
+		},
+		{
+			divider: true,
+		},
+		{
+			id: 'context-dropdown-item-document-2',
+			label: 'document 2',
+			'data-feature': 'actiondropdown.items',
+			onClick: action('document 2 click'),
+		},
+	],
+};
+
+const withComponents = {
+	id: 'context-dropdown-custom-items',
+	label: 'custom items',
+	getComponent: key => {
+		if (key === 'Action') {
+			return Action;
+		} else if (key === 'FilterBar') {
+			return FilterBar;
+		}
+		throw new Error('Component not found');
+	},
```

Comment:
Why do you pass that ?

Reply:
https://github.com/Talend/ui/pull/2937#discussion_r483093512